app/build.gradle

```diff
@@ -150,6 +150,10 @@ android {
         htmlOutput file("$project.buildDir/reports/lint/lint.html")
         disable 'MissingTranslation'
     }
+
+    buildFeatures {
+        viewBinding true
```

Comment:
🚀

Reply:
❤️ Yeah, forgot to mention that I added it for this PR/dialog and will continue to move non-conductor related views to migrate to this. Conductor-related classes will probably migrated to the new 3.0 concepts in one step :) (just a guess though)